neverbleed.c

```diff
@@ -1431,6 +1439,7 @@ int neverbleed_init(neverbleed_t *nb, char *errbuf)
 #ifdef NEVERBLEED_OPAQUE_RSA_METHOD
     rsa_default_method = RSA_PKCS1_OpenSSL();
     rsa_method = RSA_meth_dup(rsa_default_method);
+    rsa_finish = RSA_meth_get_finish(rsa_method);
```

Comment:
I may miss some context, but I don't know why this and other changes are only for newer OpenSSL since `RSA_METHOD` has `int (*finish) (RSA *rsa);` even before OpenSSL 1.1.0. If there is no specific reasons, I suggest putting `rsa_finish = rsa_default_method->finish;` in `#else` -- `#endif` below for older versions.

Reply:
Note that, while getting the `finish` function pointer from the default method and doing from a clone of it is the same, the former might be better for consistency if you approve my comment.